### PR TITLE
feat(env): adding jest

### PR DESCRIPTION
**Pull request by amirfefer**
## Updated Packages

* [ ] root
* [ ] @theforeman/builder
* [x] @theforeman/env
* [ ] @theforeman/eslint-plugin-foreman
* [ ] @theforeman/vendor
* [ ] @theforeman/vendor-dev
* [ ] @theforeman/vendor-core

## PR Type

* [ ] Bugfix
* [x] Feature
* [ ] Code style update (whitespace, formatting, missing semicolons, etc.)
* [ ] Refactoring (no functional changes, no api changes)
* [ ] Build related changes
* [ ] CI related changes
* [ ] Documentation content changes
* [ ] Other… Please describe:

## Does this PR introduce a breaking change?

* [ ] Yes
* [x] No

## Does this PR fixes open issues?

* [ ] Yes
* [x] No

## Do you use this PR in another repository?

* [ ] Yes
* [x] No

### CODE REVIEW

File: packages/env/README.md [hunk 1]
```diff
@@ -11,6 +11,7 @@
 ## What you get with this project
 
 - `@theforeman/env/babel` - Adds theforeman babel dev configuration to your project.
+- `@theforeman/env/jest` - Adds theforeman jest configuration to your project.
```
Comment (sharvit) [inline packages/env/README.md]: Notice line 16.
Wondering if we should call it `@theforeman/env/test` so one day we can replace `jest` with something else for testing.

File: packages/env/README.md [hunk 2]
```diff
@@ -38,8 +39,19 @@ module.exports = {
...
 
+
 > `@theforeman/builder/babel` will automatically load `@theforeman/env/babel` for none production environments.
 
+### install `@theforeman/env/jest`
+
+1. Add this to the `test` in `packge.json`:
+```json
+{
+  "test": "./node_modules/.bin/tfm-jest.js"
```
Comment (sharvit) [inline packages/env/README.md]: Shouldn't it suppose to be `"test": "tfm-jest"`

File: packages/env/jest/test_setup.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+require('core-js/shim');
```
Comment (sharvit) [inline packages/env/jest/test_setup.js]: You should be able to use `import` since it uses babel?

**Comment by amirfefer** [comment]
Fixed

File: packages/env/README.md [hunk 4]
```diff
@@ -11,6 +11,7 @@
 ## What you get with this project
 
 - `@theforeman/env/babel` - Adds theforeman babel dev configuration to your project.
+- `@theforeman/env/test` - Adds theforeman tests configuration to your project.
 - WIP: `@theforeman/env/lint` - Adds theforeman linting tools to you project.
 - WIP: `@theforeman/env/test` - Adds theforeman testing tools to you project.
```
Comment (sharvit) [inline packages/env/README.md]: Remove this line please. Also, IMO this description better (`Adds theforeman testing tools to you project.`)

**Comment by sharvit** [comment]
Why does this PR break the linting of vendor?

**Comment by amirfefer** [comment]
it failed due to the issue with eslint with for loops

```js
  for (const module of modules) {
    externals[module.name] = module.key;
  }
```

File: packages/env/README.md (hunk 4 above)
Comment (amirfefer) [reply packages/env/README.md]: I removed also ` WIP: @theforeman/env/lint`

**Comment by amirfefer** [comment]
tests are green :boom:

**Pull request merged**